share/spice/dogo_books/subtitle_content.handlebars

```diff
@@ -0,0 +1,4 @@
+<img src="//cdn.dogomedia.com/assets/dogobooks-a85d1c60e272132b90829f032b4f20d0.png" alt="DOGObooks"/>
```

Comment:
Let's simplify this. Please remove the custom Dogo logo and MPAA image. We can stick to plain text here and mimic the current Movies IA display:

> Runtime: 1 hr. 30 min. | PG